app/javascript/mastodon/features/ui/components/zoomable_image.tsx

```diff
@@ -306,10 +306,8 @@ export const ZoomableImage: React.FC<ZoomableImageProps> = ({
 
       <animated.img
         style={{ transform }}
-        role='presentation'
```

Comment:
While testing this change, I noticed that the image was not even being exposed correctly in the accessibility tree because of the [`role="presentation"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/presentation_role) (which effectively hides the image), so I've removed it. I'm not sure why it was being set previously.